packages/celotool/src/lib/oracle.ts

```diff
@@ -13,25 +11,18 @@ export async function installHelmChart(celoEnv: string) {
     await helmParameters(celoEnv)
   )
 }
+
 export async function removeHelmRelease(celoEnv: string) {
   await removeGenericHelmChart(releaseName(celoEnv))
 }
 
 async function helmParameters(celoEnv: string) {
-  const nodeIp = isVmBased()
-    ? await getInternalTxNodeLoadBalancerIP(celoEnv)
-    : await getPrivateTxNodeClusterIP(celoEnv)
-  const nodeUrl = `http://${nodeIp}:8545`
   return [
-    `--set celotool.image.repository=${fetchEnv(envVar.CELOTOOL_DOCKER_IMAGE_REPOSITORY)}`,
-    `--set celotool.image.tag=${fetchEnv(envVar.CELOTOOL_DOCKER_IMAGE_TAG)}`,
-    `--set mnemonic="${fetchEnv(envVar.MNEMONIC)}"`,
-    `--set oracle.cronSchedule="${fetchEnv(envVar.ORACLE_CRON_SCHEDULE)}"`,
-    `--set oracle.image.repository=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_REPOSITORY)}`,
-    `--set oracle.image.tag=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_TAG)}`,
-    `--set celocli.nodeUrl=${nodeUrl}`,
-    `--set celocli.image.repository=${fetchEnv(envVar.CELOCLI_STANDALONE_IMAGE_REPOSITORY)}`,
-    `--set celocli.image.tag=${fetchEnv(envVar.CELOCLI_STANDALONE_IMAGE_TAG)}`,
+    `--set environmentName=${celoEnv}`,
+    `--set replicas=4`,
+    `--set image.repository=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_REPOSITORY)}`,
+    `--set image.tag=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_TAG)}`,
+    `--set oracle.web3ProviderUrl=http://localhost:8545`,
```

Comment:
Should we update this to point to the full-node service?

Reply:
I'll add a follow up PR to integrate the two :)